includes/admin/class-wc-payments-admin.php

```diff
@@ -403,7 +403,7 @@ private function get_frontend_feature_flags() {
 			'paymentTimeline' => self::version_compare( WC_ADMIN_VERSION_NUMBER, '1.4.0', '>=' ),
 			'customSearch'    => self::version_compare( WC_ADMIN_VERSION_NUMBER, '1.3.0', '>=' ),
 			'accountOverview' => self::is_account_overview_page_enabled(),
-			'groupedSettings' => self::is_grouped_settings_enabled(),
```

Comment:
Moved to a new `WC_Payments_Features` class - so that `WC_Payments_Features` can be referenced elsewhere (and not just in `WC_Payments_Admin`, which is [conditionally loaded](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payments.php#L192-L194) and could cause some issues)